File: src/Server/WebApi/Dtos/CreateBookDto.cs
namespace Cbc.WebApi.Dtos;

using Cbc.WebApi.Interfaces;
using Cbc.WebApi.Models.Entities;

public class CreateBookDto : IMapTo<Book>
{
    public Guid? UpdateId { get; set; }

    public string Title { get; set; } = null!;

    public string Author { get; set; } = null!;

    public string? Description { get; set; }
}

File: src/Client/src/app/api/models/create-book-dto.ts
/* tslint:disable */
/* eslint-disable */
export interface CreateBookDto {
  author: string;
  description?: string | null;
  isbn?: string | null;
  pageCount?: number | null;
  thumbnailLink?: string | null;
  title: string;
  updateId?: string | null;
}
